### ISSUE TITLE
WordPress settings can't be queried? 

### ISSUE THREAD

**Issue by jasonbahl** [gatsbyjs/gatsby#29684]
It appears there is some sort of regression with settings from WordPress. 

Trying to query: 

```graphql
{
  wp {
    generalSettings {
      title
    }
  }
}
```

Doesn't work. It says that a `databaseId` and `uri` are required input for querying the `wp` field.

![Screen Shot 2020-11-24 at 9 57 33 AM](https://user-images.githubusercontent.com/1260765/100127862-cac8ba00-2e3c-11eb-9832-bceadbc33d2d.png)

**Comment by TylerBarnes** [comment]
@jonlow apologies for the delay here - I'm investigating this now. In the meantime if you're still running into this issue you can get around it by querying for `allWp` instead and using the only node it returns:

```
{
  allWp {
    nodes {
      allSettings {
        discussionSettingsDefaultCommentStatus
      }
    }
  }
}
```

`allWp.nodes[0].allSettings.discussionSettingsDefaultCommentStatus`

**Comment by jasonbahl** [comment]
@TylerBarnes curious why settings are considered nodes in Gatsby? Possibly that's the bug? The settings objects don't implement the Node interface. . .this seems like a bug to me. I wouldn't expect to query settings as a node

**Comment by TylerBarnes** [comment]
Everything is a node in Gatsby so this is required. When we eventually add support for multiple instances of the plugin you'll want to be able to select which site you want to get settings from too :+1:

**Comment by TylerBarnes** [comment]
```
{
  wp(url: "http://mysite.com/graphql") {
   stuffs
  }
}

**Comment by jasonbahl** [comment]
@TylerBarnes ah, interesting. Since settings objects don't have an ID, they don't follow the node spec. 

But I suppose if you are identifying them by the origin endpoint, they might indeed have an endpoint. 🤔

**Comment by TylerBarnes** [comment]
They do have an id :) it's the site url - I set it when I create each node.

**Comment by TylerBarnes** [comment]
It seems the problem here is the plugin is somehow adding the input args from another node type. I'm not sure why yet but I'll get to the bottom of it soon

**Comment by jasonbahl** [comment]
Ah, ya, since that's a departure from WPGraphQL, that should probably be documented then. Is that why we weren't able to query them individually then? Do we need to query: 

```
{
  wp(url: { eq: { "mysite.com/graphql" }) {
    ...settings
  }
}
```

**Comment by TylerBarnes** [comment]
It's undocumented right now because the plugin only supports a single instance. Using `{ wp { ...settings } }` works right now because there can only ever be 1 Wp node. Your example will be in the docs in the future but it will still break right now actually. It seems some other interface input args are somehow being applied so that databaseId and uri are required which doesn't make sense.

**Comment by TylerBarnes** [comment]
@jasonbahl @jonlow the reason is you both have types in your WPGQL schema named "Filter" and Gatsby has a bug where it then uses fields from this type to create input filter arguments for other node types resulting in broken queries.

**Comment by jasonbahl** [comment]
@TylerBarnes interesting. In my case this is the `filters` post type for documenting WPGraphQL filters: https://www.wpgraphql.com/filters/

**Comment by TylerBarnes** [comment]
This is fixed in `gatsby-source-wordpress@4.0.4`!
